climada/entity/exposures/base.py

```diff
@@ -912,9 +912,25 @@ def write_raster(self, file_name, value_name='value', scheduler=None):
         Parameters
         ----------
         file_name : str
-            name output file in tif format
+            name (and path) of the output file.
+        value_name : str or list of str
+            column(s) in the geodataframe containing data. If multiple values are provided, each column will be
+            written to a separate band in the output raster
+            The default is 'value'.
+        scheduler : str, optional
+            used for dask map_partitions.
+            “threads”, “synchronous” or “processes”
+            The default is None.
         """
-        if self.meta and self.meta['height'] * self.meta['width'] == len(self.gdf):
+        if not isinstance(value_name, list):
+            value_name = [value_name]
+
+        # Simple case: one raster band, every grid value defined in the Exposures.
+        # Then we can skip a slow points_to_raster call:
+        if self.meta and \
+                'height' in self.meta.keys() and \
+                len(value_name) == 1 and \
+                self.meta['height'] * self.meta['width'] == len(self.gdf):
```

Comment:
```suggestion
        if u_coord.grid_is_regular exp.gdf[['latitude', 'longitude']]:
```
@chahank is this alternate fix too dirty? It's not ugly

Reply:
dammit, this doesn't fix the need for meta data. ignore this.

Reply:
I am a bit confused. The point here is to fix the problem when the `latitude` and `longitude` do not correspond to the `meta` right? This here test only the `latitude` and `longitude`, not the `meta`.